apps/session-manager/src/types/notifications.ts

```diff
@@ -0,0 +1,34 @@
+import * as t from "io-ts";
+import { enumType } from "@pagopa/ts-commons/lib/types";
+import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+
+export enum NotificationMessageKindEnum {
+  "Notify" = "Notify",
+
+  "CreateOrUpdateInstallation" = "CreateOrUpdateInstallation",
+
+  "DeleteInstallation" = "DeleteInstallation",
+}
+
+export type NotificationMessageKind = t.TypeOf<typeof NotificationMessageKind>;
+export const NotificationMessageKind = enumType<NotificationMessageKindEnum>(
+  NotificationMessageKindEnum,
+  "NotificationMessageKind",
+);
+
+export enum KindEnum {
```

Comment:
I'd rather call it `DeleteInstallationKindEnum`

Reply:
Fixed in 4ede92b